sjb/config/test_cases/test_pull_request_openshift_ansible_extended_conformance_install_system_containers.yml

```diff
@@ -1,10 +1,9 @@
 ---
```

Comment:
@ingvagabund @giuseppe what branches do we care about for this job against openshift-ansible?

Reply:
IIRC 3.6+. Given we do the last three, in all branches now? @giuseppe can you verify?

Reply:
Done